frontend/src/actions/email_enroll_page.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+const getEmail = () => {
+  return dispatch => {
+    const token =
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaXNBZG1pbiI6dHJ1ZSwiaWF0IjoxNTgxMzgzNDc3LCJleHAiOjE1ODM5NzU0Nzd9.fLn5jTbyPzUMTN-h61DUQtgEdzAXUZMczGqkzFOuwT8";
+    const email = "http://localhost:1337/emails";
```

Comment:
Same comment of extracting the `http://localhost:1337` to a function or the `dotenv` package